configure.ac

```diff
@@ -1702,7 +1714,7 @@ AS_IF([$tsan],
   LDFLAGS="$SAVED_LDFLAGS"
 
   AS_CASE([$ocaml_cv_cc_vendor],
-    [gcc-[[0123456789]]-*|gcc-10-*|clang-*],
```

Comment:
It's not clear why this change is necessary, could you comment why it is needed?

Reply:
I suppose we don't expect [GCC beta 0.9 from Mach 22, 1987](https://gcc.gnu.org/releases.html).

Reply:
Yes, and it's obviously not a C99 compiler, so it really doesn't matter to handle it. But I can undo that particular change if people prefer the old version - that change is indeed not relevant to arm64 TSan support.